src/Appwrite/GraphQL/Schema.php

```diff
@@ -98,14 +98,25 @@ protected static function api(App $utopia, callable $complexity): array
             foreach ($routes as $route) {
                 /** @var Route $route */
 
-                $namespace = $route->getLabel('sdk.namespace', '');
-                $method = $route->getLabel('sdk.method', '');
-                $name = $namespace . \ucfirst($method);
+                /** @var \Appwrite\SDK\Method $sdk  */
+                $sdk = $route->getLabel('sdk', false);
 
-                if (empty($name)) {
+                if (empty($sdk)) {
                     continue;
                 }
 
+                $additionalMethods = null;
+                if (is_array($sdk)) {
+                    $mainSdk = array_shift($sdk);
+                    $additionalMethods = $sdk;
+
+                    $sdk = $mainSdk;
+                }
+
+                $namespace = $sdk->getNamespace();
+                $method = $sdk->getMethodName();
+                $name = $namespace . \ucfirst($method);
+
```

Comment:
This will hide any additional methods from graphql, we'll need to make sure both methods are available there too

Reply:
Should be fixed now: 👍